queries/html_tags/injections.scm

```diff
@@ -1,22 +1,12 @@
 ; <style>...</style>
-(
-  (style_element
-    (start_tag
-      (tag_name) .) 
-    (raw_text) @css)
-) 
-
 ; <style blocking> ...</style>
 ; Add "lang" to predicate check so that vue/svelte can inherit this
 ; without having this element being captured twice
 (
   (style_element
-    (start_tag
-      (attribute
-        (attribute_name) @_no_set_type))
-    (raw_text) @css)
-  (#not-any-of? @_no_set_type "type" "lang")
-) 
+    (start_tag) @_no_type_lang
+    (raw_text) @css
+    (#not-vim-match? @_no_type_lang "\\s(lang|type)\\s*\\="))) 
```

Comment:
Is #not-vim-match? needed? What are you exactly trying to capture that #not-any-of? "type" "lang" wasn't catching?

Reply:
Yes, sadly
`#not-any-of?` doesn't mean `only if these ... doesn't exist`, it just means, `if not any of these ... exists`

Reply:
Could a #lua-match? **not** matching those work? Would be faster..

Reply:
It will never work with lua-match, simply because lua-match doesn't support (...|...) syntax :)